docs/nconnect-get-started-cli.md

```diff
@@ -0,0 +1,154 @@
+
+Let's connect two devices using nConnect. We'll use `ping` and `ssh` to test the connection. Any two devices that can run nConnect will do: cloud server, home desktop, embedded Linux device, NAS device, mobile phone, etc.
+
+Both devices can be in different locations, home or office, cafe shop or cloud server base.
+
+The rough outline is:
+- Set up nConnect Server
+- Set up nConnect Client
+- SSH** Server from Client
+
+In this case, we assume we need to remotely access an `Ubuntu PC' at home from our `Windows laptop' wherever we are. The Ubuntu PC does not have a public IP, it is behind the Internet gateway at home. And our Windows laptop also has no public IP. We can access Ubuntu server from home, cafe, office or airport, anywhere we can connect to Internet.
+
+# Preparation
+*If you already have git and golang compiled on your machine, skip to the next section.
+
+To build nConnect from source, we need the **git** tool and the **go** compiler.
+If your PC or laptop doesn't have git and go installed, please use the following link to install them:
+
+### Install **git**
+- For Windows:
+Download and install git for Windows:
+```
+https://git-scm.com/download/win
+```
+
+- Debian/Ubuntu
+For the latest stable release for your version of Debian/Ubuntu
+```
+$ sudo apt update
+$ sudo apt install git
+```
+
+### Install **Go**
+You can download **Go** release at this URL:
+```
+https://go.dev/dl/
+```
+
+After download, please follow this URL to install **Go**
+```
+https://go.dev/doc/install
+```
+
+# Setting up the nConnect server
+To remotely access our Ubuntu PC, we first need to set up the nConnect service on this Ubuntu PC.
+Follow the steps below to install and start nConnect on your Ubuntu:
+
+- Download nConnect source code
+
+```
+$ git clone https://github.com/nknorg/nconnect
+```
+
+- Build nConnect
+
+```
+$ cd nConnect
+$ make
+```
+After **make** it should create **nConnect** executable in the current folder.
+
+- Get nConnect server address:
+nConnect Client connects to nConnect Server using the server's address. So first we get the server address as follows:
+```
+$ ./nConnect -s --address
+nConnect.l608io.dfc5a05ae4c5f05e4db0f2......522cc90d523a0754fa97
+```
+Copy the string leady through **nConnect.**, this is the address of the server. For security, I use **......** to hide address details. if you copy or paste it, you should keep original data.
+
+- Start nConnect Server:
+Now it's time to start the nConnect server:
+```
+$ ./nConnect -s 
+```
+Great, the nConnect server is started.
+
+# Set Up nConnect Client
+Now we set up the nConnect client on the Windows laptop.
+Just like the server, we download the nConnect source code from github:
+
+- Download nConnect source code
+
+```
+$ git clone https://github.com/nknorg/nconnect
+```
+
+- Build nConnect
+
+```
+$ cd nconnect
+$ go build -o nConnect.exe bin/main.go
+```
+After **go build** it should create **nConnect.exe** executable in the current folder.
```

Comment:
Same here

Reply:
Sorry, nconnect-get-started-cli.md is removed. because its content are moved into nconnect-get-started.md and nconnect-build-from-source-code.md